reactivesocket-cpp/src/Frame.h

```diff
@@ -5,15 +5,15 @@
 #include <iosfwd>
 #include <limits>
 
-#include <folly/Optional.h>
+/// Needed for inline d'tors of frames.
+#include <folly/io/IOBuf.h>
+#include <folly/io/IOBufQueue.h>
```

Comment:
that looks like some cpp file needs to include logging.h
its ok. we can fix this in another diff